examples/systemcalls/systemcalls.c

```diff
@@ -11,12 +36,25 @@ bool do_system(const char *cmd)
 {
 
 /*
- * TODO  add your code here
+ * TODO : add your code here
  *  Call the system() function with the command set in the cmd
  *   and return a boolean true if the system() call completed with success 
  *   or false() if it returned a failure
 */
 
+    int returnval;
+    
+    openlog("AESD A3 Part 1 - Do System routine", LOG_PID, LOG_USER);
+
+    returnval = system(cmd);
+    
+    if (returnval == -1)
```

Comment:
@tanmay-mk  think about what would happen if the cmd returned with an error code other than -1 here.